src/main/groovy/com/google/protobuf/gradle/internal/AndroidExt.groovy

```diff
@@ -0,0 +1,71 @@
+package com.google.protobuf.gradle.internal
+
+import com.android.build.gradle.api.BaseVariant
+import com.android.build.gradle.api.TestVariant
+import com.android.build.gradle.api.UnitTestVariant
+import com.android.builder.model.SourceProvider
+import com.google.protobuf.gradle.ProtobufExtension
+import com.google.protobuf.gradle.Utils
+import com.google.protobuf.gradle.tasks.ProtoSourceSet
+import groovy.transform.CompileDynamic
+import org.gradle.api.Project
+import org.gradle.api.artifacts.Configuration
+import org.gradle.api.attributes.Attribute
+import org.gradle.api.attributes.AttributeContainer
+import org.gradle.api.file.CopySpec
+import org.gradle.api.tasks.SourceTask
+import org.gradle.language.jvm.tasks.ProcessResources
+
+@CompileDynamic
+class AndroidExt {
```

Comment:
Add comment: `Temporarily hack. TODO: Move all AndroidExt to AndroidProtobufPlugin`